blackhole.go

```diff
@@ -46,6 +46,11 @@ func (client *BlackholeClient) PublishAuditLog(auditLogBuilder *AuditLogBuilder)
 	return nil
 }
 
+func (client *BlackholeClient) PublishAuditLogToTopic(topic string, auditLogBuilder *AuditLogBuilder) error {
```

Comment:
This interface is confusing. Why does an audit need to be published on another topic? We educated the consumer that they do not need to care about the topics, but now you introduce new options.

Reply:
> This interface is confusing. Why does an audit need to be published on another topic? We educated the consumer that they do not need to care about the topics, but now you introduce new options.

This is for account history, this should be another topic, I used to think to use PublishAccountHistory as the function, but I think I may not be so common to put it into the lib @angelo-chan